source/connection-strings/mongodb-database-connection-strings.txt

```diff
@@ -1,3 +1,126 @@
+.. _rm-mongodb-database-connection-strings:
+
 ===================================
 MongoDB Database Connection Strings
 ===================================
+
+.. include:: /includes/uri-usage-disclaimer.rst
+
+To start a migration job, Relational Migrator must connect to your MongoDB 
+database. Relational Migrator requires a username and password which has ``readWrite`` 
+access in the target database. This page describes 
+the procedure to make a authenticated user account and the Uniform Resource 
+Identifier (URI) formats for defining connections to your target 
+MongoDB database. 
+
+Relational Migrator can use both Atlas and On-Premises URIs. This page 
+provides separate instructions for each deployment type.
+
+Atlas
+-----
+
+Connect to Your Deployment
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To provision user accounts in Atlas you need to be 
+`logged in to the Atlas GUI <https://cloud.mongodb.com/v2/>`__.
+
+Create a New User
+~~~~~~~~~~~~~~~~~
+
+#. In the :guilabel:`Security` section of the left navigation, click :guilabel:`Database Access`.
+#. Click :guilabel:`Add New Database User`.
+#. In the Authentication Method section of the :guilabel:`Add New Database User` modal window, select the box labeled :guilabel:`Password`.
+#. Under :guilabel:`Password Authentication`, enter the username ``migrator-service`` for the new user in the top text field.
+#. Enter the password ``password`` for the new user in the lower text field.
+#. Under :guilabel:`Database User Privileges`, click :guilabel:`Built-in Role`.
+#. Select :guilabel:`Read and write to any database`.
+#. Click :guilabel:`Add User`
+
+Connection URI
+~~~~~~~~~~~~~~
+
+Using the previously created account credentials, format the connection
+URI for the target database. In this case ``MongoEnterprises``.
+
+.. code-block::
+
+   mongodb+srv://migrator-service:password@sandbox.xxxxx.mongodb.net/MongoEnterprises
+
+On-Premises
+-----------
+
+In this example, use ``mongosh`` to provision a user account that 
+connects to the ``MongoEnterprises`` database.
+
+This examples assume your deployment is running on ``localhost`` and the 
+default port of ``27017``.
+
+Connect to Your Deployment
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
(repetition) 
Consider slight rewrite. Something like:
"When connecting to your deployment, your user account requires the ..."